### DISCUSSION TITLE
IBKR Trailing Stop failure on restart

### DISCUSSION THREAD

**Discussion by matheosz1** [nautechsystems/nautilus_trader#1275] [Q&A]
Hi! I was curious if it's normal behaviour that nautilus trader fails to restart the trading node when a trailing stop order is open in interactive broker. Here is the error: <img width="755" alt="Screenshot 2023-10-16 at 8 00 13 AM" src="https://github.com/nautechsystems/nautilus_trader/assets/104100249/228156bc-4ecd-4b96-b4a5-ce5ef04eccb5">

Am I doing something wrong? Should I be closing them in my strategy? For now I've been closing them manual with the obvious issue of a difference between that and my backtests.

**Comment by rsmb7z** [comment]
@matheosz1 no this is not normal and it should restart without any issue, I will fix it. Thanks for reporting.
